tests/system/general/test_45_FilterSolutions/test_filter/test_attributes.py

```diff
@@ -151,7 +151,7 @@ def test_minimum_order_group_delay_cutoff(self):
         design.attributes.minimum_order_group_delay_cutoff = "500 MHz"
         assert design.attributes.minimum_order_group_delay_cutoff == "500 MHz"
 
-    def test_minimum_order_stop_band_freq(self):
+    def test_minimum_order_stop_band_freq_second_time(self):
```

Comment:
@ramin4667 I just renamed the test. In this file it is defined twice if you look few lines above.
I was not sure if it was done on purpose to maybe set up again some parameters.